README.md

```diff
@@ -13,13 +13,15 @@ Our templates offer a rich boilerplate to jump start Go Gin-based application de
 ## Get Started
 
 ### Prerequisite 
+
 - [Python interpreter](https://docs.python.org/3/using/index.html)
 - Cookiecutter
   - Mac: `brew install cookiecutter`
-  - Ubuntu: `brew install cookiecutter`
+  - Debian/Ubuntu: `sudo apt-get install cookiecutter`
 - Go version >= 1.15
```

Comment:
If you want to show that it's compatible with Go 1.15 and 1.6, then matrix testing is needed. Otherwise, this must be changed to:

```suggestion
- Go version >= 1.16
```

Reply:
Fixed in afc028bc548644e56186f9786d6e1c1c5c95caf7